src/translate.js

```diff
@@ -33,12 +24,28 @@ export default (singleton) => {
     finalOptions = {
       ...defaultOptions,
       ...finalOptions,
+      replacementsContext:  (finalOptions && finalOptions.context) || 'no_context', 
```

Comment:
**Question:** Wouldn't it be good to reuse the already existing instead of saving a secondary replacement context? 

to my understanding the replacementContext is always the same as the original translation context?

**Question 2:** When thinking about contexts, wouldn't that be a good way to handle replacements without having the secondary JSON file? 

I mean you could list 2 translations:

```
msgctxt "old"
msgid "go back"
msgstr "go back old

msgctxt "new"
msgid "go back"
msgstr "go back new"

msgid "hi"
msgstr "Hello"
```

And then pick the correct translations via the contexts instead of using custom replacement files for each applications.

Reply:
> **Question:** Wouldn't it be good to reuse the already existing instead of saving a secondary replacement context?

The context itself is always the same, however, we need a way to specify it in the `replacements.json`.
We can't use an empty string  as a key for replacements without context because it is not a valid object key. 
That's why there is a `no_context` key in the `replacements.json` for this case.

> **Question 2:** When thinking about contexts, wouldn't that be a good way to handle replacements without having the secondary JSON file?
It won't fit because of two reasons:
1) In that case the original context will be lost
2) Translation team/service looks at the `msgid` of the `message.pot`. Having the same `msgid` will require to change the translation process for them to translate it to the different languages.